services/workflows-service/docker-compose.yml

```
@@ -63,3 +78,9 @@
       retries: 10
 volumes:
   postgres15: ~
+  caddy_config:
+  caddy_data:
+networks:
+  mynetwork:
+    external: true
+    name: host
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)